docs/content/docs/testing/telemetry.md

```diff
@@ -240,24 +242,54 @@ export class ConsoleLogger implements ITelemetryBaseLogger {
 }
 ```
 
-This custom logger is then created and passed into the `getContainer()` and `createContainer()` in a `TinyliciousClient`
-object. The custom logger is now hooked up to the container's telemetry system.
+This custom logger is then created in the `azureConfig` before being passed into `AzureClient`
+constructor. The custom logger is now hooked up to the container's telemetry system.
 
 ```ts
 async function start(): Promise<void> {
-    // Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
-    // and hook to the Telemetry system
-    const consoleLogger: ConsoleLogger = new ConsoleLogger();
-
-    // Get or create the document depending if we are running through the create new flow
-    const client = useAzure ? AzureClient :  new TinyliciousClient();
-    const [fluidContainer, containerServices] = createNew
-        ? await client.createContainer({ id: containerId, logger: consoleLogger }, containerSchema)
-        : await client.getContainer({ id: containerId, logger: consoleLogger }, containerSchema);
+  // Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
+  // and hook to the Telemetry system
+  const azureConfig = {
+      connection: connectionProps,
+      logger: new ConsoleLogger(),
+  };
+  const client = new AzureClient(azureConfig);
+  let container: FluidContainer;
+  let services: AzureContainerServices;
+  let id: string;
```

Comment:
nit: unused here

Reply:
Good point. I was thinking about copying the entire code snippet as is to maintain it's originality, but I realized the readers probably don't know where this code is originally from, so better to just focus on what's relevant 😅